Makefile

```diff
@@ -0,0 +1,15 @@
+PROTO_FILE=api/githubsearch.proto
```

Comment:
If there is a Makefile, there should be targets to do everything needed in the project, like running the server, running the client, installing dependencies, etc.